### PR TITLE
License Manager Changes

### PR DESCRIPTION
**Pull request by Gurmeenkour99**
This PR need to be tested with License Manger PR for the integration purpose . In this PR client_role is added in the CURL request.
https://github.com/ladybirdweb/agora-license-manager/issues/178

### CODE REVIEW

**Review by sowmiyasubramaniyan — DISMISSED** [review]

File: app/Http/Controllers/License/LicenseController.php [hunk 1]
```diff
@@ -276,7 +276,8 @@ public function createNewLicene($orderid, $product, $user_id, $licenseExpiry, $u
             $productId = $this->searchProductId($sku);
             $OauthDetails = $this->oauthAuthorization();
             $token = $OauthDetails->access_token;
-            $addLicense = $this->postCurl($url.'api/admin/license/add', "api_key_secret=$api_key_secret&product_id=$productId&license_code=$serial_key&license_require_domain=1&license_status=1&license_order_number=$orderNo&license_domain=$domain&license_ip=$ip&license_require_domain=$requireDomain&license_limit=6&license_expire_date=$licenseExpiry&license_updates_date=$updatesExpiry&license_support_date=$supportExpiry&license_disable_ip_verification=0&license_limit=1", $token);
+            $addLicense = $this->postCurl($url.'api/admin/license/add', "api_key_secret=$api_key_secret&product_id=$productId&license_code=$serial_key&license_require_domain=1&license_status=1&license_order_number=$orderNo&license_domain=$domain&license_ip=$ip&license_require_domain=$requireDomain&license_limit=6&license_expire_date=$licenseExpiry&license_updates_date=$updatesExpiry&
+            license_client_role='admin'&license_support_date=$supportExpiry&license_disable_ip_verification=0&license_limit=1", $token);
```
Comment (sandesh556) [inline app/Http/Controllers/License/LicenseController.php:280]: why are we adding client role to this API, the end point doesn't even accept it

**Review by sandesh556 — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/License/LicenseController.php [hunk 2]
```diff
@@ -276,7 +276,8 @@ public function createNewLicene($orderid, $product, $user_id, $licenseExpiry, $u
             $productId = $this->searchProductId($sku);
             $OauthDetails = $this->oauthAuthorization();
             $token = $OauthDetails->access_token;
-            $addLicense = $this->postCurl($url.'api/admin/license/add', "api_key_secret=$api_key_secret&product_id=$productId&license_code=$serial_key&license_require_domain=1&license_status=1&license_order_number=$orderNo&license_domain=$domain&license_ip=$ip&license_require_domain=$requireDomain&license_limit=6&license_expire_date=$licenseExpiry&license_updates_date=$updatesExpiry&license_support_date=$supportExpiry&license_disable_ip_verification=0&license_limit=1", $token);
+            $addLicense = $this->postCurl($url.'api/admin/license/add', "api_key_secret=$api_key_secret&product_id=$productId&license_code=$serial_key&license_require_domain=1&license_status=1&license_order_number=$orderNo&license_domain=$domain&license_ip=$ip&license_require_domain=$requireDomain&license_limit=6&license_expire_date=$licenseExpiry&license_updates_date=$updatesExpiry&
```
Comment (sandesh556) [inline app/Http/Controllers/License/LicenseController.php:279]: if there no change done here, revert these changes

**Review by sandesh556 — CHANGES_REQUESTED** [review]

**Comment by SarfrazS** [comment]
@Gurmeenkour99 What are the modules and functionality needed to test in this PR?

**Comment by SarfrazS** [comment]
Dependency PR License Manager https://github.com/ladybirdweb/agora-license-manager/pull/177 Migration Not working once working then will start testing this pr.

**Review by SarfrazS — APPROVED** [review]
Working fine.

**Pull request merged**